flexmeasures/api/v2_0/implementations/sensors.py

```diff
@@ -81,18 +81,18 @@ def post_price_data_response(  # noqa C901
                 ea = parse_entity_address(market, entity_type="market")
             except EntityAddressException as eae:
                 return invalid_domain(str(eae))
-            market_id = ea["sensor_id"]
+            sensor_id = ea["sensor_id"]
```

Comment:
Will this crash if the entity address was a fm0 one?

Reply:
API v2.0 is incompatible with fm0, and line 81 raises an `EntityAddressException` if you try it anyway.